README.md

```diff
@@ -82,6 +83,13 @@ You also need to download the Appium client for your language so you can write t
 * Windows 10
 * [Documentation](/docs/en/appium-setup/running-on-windows.md)
 
+#### Mac Requirements
+
+* Mac OS X 10.7 +
+* The Appium for Mac binaries do not currently ship with Appium, so you will need to:
```

Comment:
I think these instructions should go into the setup docs and not the main README. In this particular place I think it's too much detail for setup instructions, and people are more likely to confuse "mac requirements" with "ios requirements". I think mac app automation could have its own section in the osx setup doc file?